playbook.yml

```diff
@@ -1,54 +1,68 @@
 - hosts: localhost
   gather_facts: no
   tasks:
+    - name: Run Workload
+      block:
+        - include_role:
+            name: "uuid"
 
-  - name: Run Workload
-    block:
-    - include_role:
-        name: "uuid"
-
-    - include_role:
-        name: "uperf-bench"
-      vars:
-        uperf: "{{ workload.args }}"
-      when: workload.name == "uperf"  and workload.args.pair > 0
-
-    - include_role:
-        name: "iperf3-bench"
-      vars:
-        iperf3: "{{ workload.args }}"
-      when: workload.name == "iperf3" and workload.args.pairs > 0
-
-    - include_role:
-        name: sysbench
-      vars:
-        sysbench : "{{ workload.args }}"
-      when: workload.name == "sysbench" and workload.args.enabled is defined and workload.args.enabled
-
-    - include_role:
-        name: "ycsb-bench"
-      vars:
-        ycsb: "{{ workload.args }}"
-        ycsb_load: "{{ workload.args.loaded | default(false) }}"
-      with_items: '{{ workload.args.workloads }}'
-      when: workload.name == "ycsb" and workload.args.workloads | length > 0
-
-    - include_role:
-        name: "byowl"
-      vars:
-        byowl: "{{ workload.args }}"
-      when: workload.name == "byowl" and workload.args.clients > 0
-
-    - include_role:
-        name: "fio-distributed"
-      vars:
-        fiod: "{{ workload.args }}"
-      when: workload.name == "fio_distributed" and workload.args.servers > 0
-
-    - include_role:
-        name: "pgbench"
-      vars:
-        pgbench: "{{ workload.args }}"
-      when: workload.name == "pgbench" and workload.args.clients > 0
-
-    when: workload is defined
+        - include_role:
+            name: "uperf-bench"
+          vars:
+            uperf: "{{ workload.args }}"
+          when: workload.name == "uperf"  and workload.args.pair > 0
+
+        - include_role:
+            name: "iperf3-bench"
+          vars:
+            iperf3: "{{ workload.args }}"
+          when: workload.name == "iperf3" and workload.args.pairs > 0
+
+        - include_role:
+            name: "fio-bench"
+          vars:
+            fio: "{{ workload.args }}"
+          when: workload.name == "fio" and workload.args.clients > 0
+
+        - include_role:
+            name: sysbench
+          vars:
+            sysbench: "{{ workload.args }}"
+          when: workload.name == "sysbench" and workload.args.enabled is defined and workload.args.enabled
+
+        - include_role:
+            name: "ycsb-bench"
+          vars:
+            ycsb: "{{ workload.args }}"
+            ycsb_load: "{{ workload.args.loaded | default(false) }}"
+          loop:
```

Comment:
not a neat rebase as the loop control was changed, can you please make sure that there is no other change in here other than adding smallfile. i.e. remove the spacing change as its not necessary.